compiler/rustc_codegen_cranelift/src/vtable.rs

```diff
@@ -41,7 +38,7 @@ pub(crate) fn min_align_of_obj(fx: &mut FunctionCx<'_, '_, '_>, vtable: Value) -
         pointer_ty(fx.tcx),
         vtable_memflags(),
         vtable,
-        (ALIGN_INDEX * usize_size) as i32,
+        (ty::COMMON_VTABLE_ENTRIES_SIZE * usize_size) as i32,
```

Comment:
This should have been `_ALIGN` not `_SIZE`. This caused a miscompilation in https://github.com/bjorn3/rustc_codegen_cranelift/issues/1179.

Reply:
Oops, sorry...